<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translater</class>
 <widget class="QWidget" name="Translater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>390</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string> 极简翻译</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/icons/ya.svg</normaloff>:/icons/icons/ya.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="TextFrom">
     <property name="toolTip">
      <string>输入你想翻译的文本</string>
     </property>
     <property name="plainText">
      <string>English</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="TextTo">
     <property name="toolTip">
      <string>这里将显示翻译结果</string>
     </property>
     <property name="plainText">
      <string>英语</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pBtnCopyTranlate">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
       <property name="toolTip">
        <string>复制翻译结果</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/icons/copy.png</normaloff>:/icons/icons/copy.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="ENTOZH">
       <property name="toolTip">
        <string>翻译为中文</string>
       </property>
       <property name="text">
        <string>zh</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="ZHTOEN">
       <property name="toolTip">
        <string>翻译为英文</string>
       </property>
       <property name="text">
        <string>en</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="isFix">
       <property name="toolTip">
        <string>不自动隐藏窗口</string>
       </property>
       <property name="text">
        <string>fix</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
